.github/workflows/adhoc.yml

```
@@ -97,4 +102,19 @@
       - name: Test Docker image
         run: |
           echo "Testing Docker image..."
-          ./scripts/test_docker_image.sh -t "branch-${{ steps.version.outputs.safe_branch }}"
+          # Only verify version and core functionality without starting the full app
+          # Use the --basic flag to avoid requiring environment variables
+          ./scripts/test_docker_image.sh -t "sha-${{ steps.version.outputs.short_sha }}" --basic --discord-token "${{ secrets.FAKE_DISCORD_TOKEN }}"
+
+          echo "Docker image test passed successfully!"
+
+      - name: Push Docker image with branch tags
+        uses: docker/build-push-action@v4
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)